meet-data/ipf/ipf-import

```diff
@@ -8,49 +8,106 @@ set -e
 SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 REPOSCRIPTDIR="${SCRIPTDIR}/../../scripts"
 
-if [ $# -ne 2 ]; then
+if [ $# -lt 2 ]; then
 	echo "Usage: $0 dirname URL"
 	exit 1
 fi
 
 mkdir "$1"
 cd "$1"
 
-# Download the results page.
-wget --output-document=results.html "$2"
-cp results.html results.xls
+#One file
+if [ $# -eq 2 ];then
+	# Download the results page.
+	wget --output-document=results.html "$2"
+	cp results.html results.xls
 
-# Save the website URL for future use, since it's pretty hard to tell
-# what meets on the site are tracked and which aren't.
-echo "$2" > URL
+	# Save the website URL for future use, since it's pretty hard to tell
+	# what meets on the site are tracked and which aren't.
+	echo "$2" > URL
 
-# Replace any commas in the xls file with periods.
-# Commas are used as decimal points in the European style.
-sed -i -e 's/,/\./g' results.xls
 
-# Strikethrough HTML is used to denote failures.
-# Replace <s> with '-' and remove </s>.
-sed -i -e 's/<s>/-/g' results.xls
-sed -i -e 's/<\/s>//g' results.xls
+	# Replace any commas in the xls file with periods.
+	# Commas are used as decimal points in the European style.
+	sed -i -e 's/,/\./g' results.xls
 
-# Use LibreOffice to automatically convert the <table> to a csv file.
-# This creates results.csv.
-libreoffice --headless --convert-to csv results.xls
-rm results.html results.xls
+	# Strikethrough HTML is used to denote failures.
+	# Replace <s> with '-' and remove </s>.
+	sed -i -e 's/<s>/-/g' results.xls
+	sed -i -e 's/<\/s>//g' results.xls
 
-# ASCII 0xa0 (a space) creeps in everywhere for no good reason.
-sed -i -e 's/\xa0//g' results.csv
+	# Use LibreOffice to automatically convert the <table> to a csv file.
+	# This creates results.csv.
+	libreoffice --headless --convert-to csv results.xls
+	rm results.html results.xls
 
-# Remove redundant spacing.
-sed -i -e 's/  / /g' results.csv
-sed -i -e 's/, /,/g' results.csv
-sed -i -e 's/ ,/,/g' results.csv
+	# ASCII 0xa0 (a space) creeps in everywhere for no good reason.
+	sed -i -e 's/\xa0//g' results.csv
 
-# Failures are denoted as "X", when we just want a blank space.
-sed -i -e 's/,X,/,,/g' results.csv
+	# Remove redundant spacing.
+	sed -i -e 's/  / /g' results.csv
+	sed -i -e 's/, /,/g' results.csv
+	sed -i -e 's/ ,/,/g' results.csv
+
+	# Failures are denoted as "X", when we just want a blank space.
+	sed -i -e 's/,X,/,,/g' results.csv
+
+	#Skips are sometimes denotes as "#"
+	sed -i -e 's/#//g' results.csv
+
+
+	# Disqualifications use ascii 0x97, a long dash.
+	sed -i -e 's/\x97/DQ/g' results.csv
+
+
+
+else #two or more files
+	for (( ii=2; ii<=$# ; ii++))
```

Comment:
Surely this can handle the `$#=2` case also?

Reply:
I've separated the $# =2 and $# > 2 cases just so that the results files can be labelled differently, although it does mean a lot of code is repeated for a small amount of functionality. I can change this if you want?

Reply:
If it's not too much work to avoid the repetition, that would be preferable.

Reply:
Removed in the latest commit.